x-pack/plugins/cloud_security_posture/public/pages/findings/findings_flyout/findings_flyout.tsx

```diff
@@ -53,28 +64,32 @@ interface FindingFlyoutProps {
   findings: CspFinding;
 }
 
-const Cards = ({ data }: { data: Card[] }) => (
-  <EuiFlexGrid direction="column" gutterSize={'l'}>
-    {data.map((card) => (
-      <EuiFlexItem key={card.title} style={{ display: 'block' }}>
-        <EuiCard textAlign="left" title={card.title} hasBorder>
-          <EuiDescriptionList
-            compressed={false}
-            type="column"
-            listItems={card.listItems.map((v) => ({ title: v[0], description: v[1] }))}
-            style={{ flexFlow: 'column' }}
-            descriptionProps={{
-              style: { width: '100%' },
-            }}
-          />
-        </EuiCard>
-      </EuiFlexItem>
-    ))}
-  </EuiFlexGrid>
-);
+const Cards = ({ data }: { data: Card[] }) => {
+  const { euiTheme } = useEuiTheme();
```

Comment:
nit euiTheme is it used?

Reply:
was not! good catch. was messing with it to solve the sizing problem but ended up with another solution. removed.